examples/react/src/App.js

```diff
@@ -391,6 +405,26 @@ renderLoginButtons() {
           logoStyle={{marginLeft:0}}
           onClick={()=>this.handleLogin("lynx")}
       />
+      <LoginButton provider='portis'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("portis")}
+      />
+      <LoginButton provider='whalevault'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("whalevault")}
+      />
+      <LoginButton provider='simpleos'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("simpleos")}
+      />
+      <LoginButton provider='keycat'
+          buttonStyle={{width:250, marginTop:'24px'}}
+          logoStyle={{marginLeft:0}}
+          onClick={()=>this.handleLogin("keycat")}
```

Comment:
I get "Cannot initiate a session, invalid wallet provider or wallet provider ID was passed" for keycat. Pretty sure I have 3.1.2 of eos-transit and the right transit providers installed. Do you get this same error?

Reply:
Yes same error for me